capture/listener.go

```diff
@@ -526,7 +520,7 @@ func (t *Listener) readPcap() {
 					continue
 				}
 
-				dataOffset := (data[12] & 0xF0) >> 4
+				dataOffset := data[12] >> 4
```

Comment:
A lot of magic  :) 
Can you explain what did\does it mean?

Reply:
in TCP Header the data offset is identified at `the last 4 bits of 12th byte` thats why we've done this shifting